### PR TITLE
fix: Update IdeCard.js

### PR DESCRIPTION
**Pull request by s100**
Contributes to #2213

#### What did you change?

Added alt text to the "opens in new tab" icon on `<IdeCard>`s of type `'externalLink'`.

#### How did you test and verify your work?

Manual verification, plus CI.

### CODE REVIEW

File: packages/cdai/src/components/IdeCard/IdeCard.js [hunk 1]
```diff
@@ -294,6 +294,7 @@ export class IdeCard extends React.Component {
             <div
               className={`${idePrefix}-card--tile-img`}
               {...idAttribute(`ide-card-${cardId}-launch-Image`)}
+              alt="Opens in new tab"
```
Comment (davidmenendez) [inline packages/cdai/src/components/IdeCard/IdeCard.js:297]: we shouldn't have any hard coded strings for translation purposes. this really should be sent as a prop.

**Review by davidmenendez — APPROVED** [review]

**Review by davidmenendez — CHANGES_REQUESTED** [review]
one small comment

File: packages/cdai/src/components/IdeCard/IdeCard.js (hunk 1 above)
Comment (s100) [reply packages/cdai/src/components/IdeCard/IdeCard.js]: @davidmenendez Makes sense. I have updated the component to have a new optional string prop `externalLinkText`, which is passed down through `renderCardContent` and `renderCardHeader` to this point. Currently I'm retaining the English string "Opens in new tab" as the *default* value of this prop, as I see is done in a few other places in this codebase.

**Review by davidmenendez — APPROVED** [review]
thanks!

**Pull request merged**